src/domain/community/user/user.resolver.mutations.ts

```diff
@@ -214,4 +229,47 @@ export class UserResolverMutations {
     );
     return await this.userAuthorizationService.applyAuthorizationPolicy(user);
   }
+
+  private async assignUserToOrganizationByDomain(
+    agentInfo: AgentInfo,
+    user: IUser
+  ): Promise<void> {
+    if (!agentInfo.emailVerified) {
+      return;
```

Comment:
I think this is an exceptional case, the user should _not_ be able to reach that point - silently returning is not correct in that case. An exception should be thrown.